packages/augur-ui/src/modules/auth/actions/update-sdk.ts

```diff
@@ -20,10 +20,9 @@ export const updateSdk = (
 
   try {
     dispatch(updateAppStatus(Ox_ENABLED, augurSdk.sdk.zeroX));
-
-    // TODO: useGnosis should always be true or fetchable from
     // the SDK -- maybe augurSdk.sdk.getUseGnosisSafe() ??
-    const useGnosis = window.localStorage.getItem('isGnosis');
+    // Overwrite to disable gnosis
+    const useGnosis = true;
```

Comment:
This should really come from the sdk -- we can configure it at the highest level there and pass in that we're gonna use it from the config.